slybot/slybot/plugins/selectors/__init__.py

```diff
@@ -0,0 +1,28 @@
+
+class Selectors(object):
+    def setup_bot(self, settings, spec, items, extractors):
+        self.selectors = {} # { item_type: { field_name: {..} }
+
+        for template in spec['templates']:
+            selectors = self.selectors.setdefault(template['scrapes'], {})
+            selectors.update(template.get('selectors', {}))
```

Comment:
This only allows a single css/xpath expression per field name per item type, right?